section1/reflection.md

```diff
@@ -1,19 +1,39 @@
 ## Section 1 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+I tend to wait too long. Asking questions causes me a *lot* of anxiety, for all the reasons listed in the articles. These are some of my personal goals for this program: get more comfortable not knowing things, not being "the best", and asking questions that might reveal all of the above.
 
 2. What are the data types you learned about in this section? In your own words, define each.
+## Primitive data types:
+* string: consists of letters, numbers, other symbols, enclosed in ' ' or " ". Is protected in that it won't be changed by any equations. 
+* number: consists of numbers only, but can be positive, negative, or floating integers (0.00)
+* Booleans: consists only of true or false
+
 
 3. How would you log the string `"Hello World!"` to the console?
+console.log("Hello, World!");
 
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+//
+or
+/*
+Comments are used to prevent what you type from being "run" with the rest of the code. This might be because you have written pseudocode or explanations of your code, or it might be because you have bits of code you don't want run at the moment.
 
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
+A variable is something that holds data. I'd say: "You know how everyone has different stats, like different favorite colors, ages, heights, names, etc.? Those are all variables about people."
+
+
 
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+I use Airbnb constantly. Airbnb probably uses variables for home type (apartment, RV, hotel, house, etc., and these are likely strings), number of bedrooms (likely number data types), and then maybe the calendar uses Booleans to track whether a given home is available on a given date.
 
 7. In your own words, explain what concatenation is.
+Concatenation is joining strings together using + .
 
 8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
 
+Well, probably google uses concatenation when I do a search:
+**Thai food in Eugene** is probably concatenated regardless of whether I do it in the search bar.
+
 9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+I still need some help troubleshooting and understanding why things go wrong when they do. I'm sure this will come! Oh, and another question is about type coercion - I have previously coded a **tiny** bit with Python, and it didn't seem like Python did type coercion, since we had to manually change data types. Is that correct? Why do some languages include type coercion and some don't?
```

Comment:
Great question! JavaScript is a little quirky in that it performs type coercion without explicit instruction. As far as I've learned, the 'why' mostly goes back to early decisions made by the language's developers, but I'd point you to Google if you're interested in a deeper understanding 🔍 

Like you know from Python though, _most_ language require explicit type coercion.